crates/bevy_winit/src/lib.rs

```diff
@@ -73,7 +73,7 @@ fn change_window(_: &mut World, resources: &mut Resources) {
                 }
                 bevy_window::WindowCommand::SetResolution { width, height } => {
                     let window = winit_windows.get_window(id).unwrap();
-                    window.set_inner_size(winit::dpi::LogicalSize::new(width, height));
+                    window.set_inner_size(winit::dpi::PhysicalSize::new(width, height));
```

Comment:
This should be `LogicalSize`. WindowDescriptor.width + WindowDescriptor.height are "logical size" (which is desirable). This should mirror that.

Reply:
internally this is still the physical size, but it is calculated from the logical size specified by the use in the call to `set_resolution`

Reply:
ah yeah you're right. carry one :smile: